learning.py

```diff
@@ -369,21 +368,21 @@ def plurality_value(examples):
         return DecisionLeaf(popular)
 
     def count(attr, val, examples):
-        "Count the number of examples that have attr = val."
+        """Count the number of examples that have attr = val."""
         return len(e[attr] == val for e in examples) #count(e[attr] == val for e in examples)
 
     def all_same_class(examples):
-        "Are all these examples in the same target class?"
+        """Check if all the examples are in the same target class"""
```

Comment:
I think the original wording is better, because it makes it clear the answer is True or False. "Check if" does not make that as clear.